### ISSUE TITLE
Bug / Cannot read property 'disconnect' of undefined at SelectWrapper.componentDidUnload (1.5.2)

### ISSUE THREAD

**Issue by leifmarcus** [porscheui/porsche-design-system-contribution#43]
**Description**  
when a PSelectWrapper Component is unloaded it will throw an error, because `this.selectObserver` can be undefined in some cases. As the observer is only created if the device is not a touch device `if (!this.isTouch) { ... this.observeSelect() }` this will always throw an error on touch devices.

**To Reproduce**  
1. create a react application
2. add a PSelectWrapepr as follows

```jsx
<PSelectWrapper label={"Some label"} hideLabel={false}><select name={"some-name"}>
    <option value={""}>Select a country</option>
    <option value={"AF"}>Afghanistan</option>
    <option value={"AX"}>Åland Islands</option>
    <option value={"AL"}>Albania</option>
    <option value={"DZ"}>Algeria</option>
</select></PSelectWrapper>
```

3. make sure, that componentDidUnload is called by removing the component from the ReactDOM.
4. See error inside the developer tools console.

**Expected behavior**  
There should not be a runtime error.

**Screenshots**  
![image](https://user-images.githubusercontent.com/2342671/88385523-cd43d600-cdae-11ea-830e-58d6d3c4d1b5.png)
![image](https://user-images.githubusercontent.com/2342671/88385680-18f67f80-cdaf-11ea-8524-a7ceb439b8d1.png)

**Test Device (please complete the following information):**  
 - Device: Desktop, Tablet
 - OS: Mac OS Catalina (10.15.5), iOS 13.6
 - Browser chrome, iOS Safari
 - Version chrome: 84.0.4147.89

**Comment by leifmarcus** [comment]
In Version 1.5.3 this bug is still there.  `this.selectObserver` is still undefined in the `componentDidUnload` function.

![image](https://user-images.githubusercontent.com/2342671/90525727-da66a180-e16f-11ea-90a9-9bc8bad69801.png)

This error happens always, when navigating to a page (with react router) that contains PSelectWrapper components, not when open a page directly.

**Comment by mhaegeleMHP** [comment]
We published a release candidate v1.5.5-rc.0 which should fix the issue for you. Could give us feedback on this? @leifmarcus

**Comment by marcelbertram** [comment]
Released in 1.5.5.
